common/lib/xmodule/xmodule/util/duedate.py

```diff
@@ -21,3 +21,15 @@ def get_extended_due_date(node):
     if not extended or extended < due_date:
         return due_date
     return extended
+
+
+def escape_export_name(name, replace_with='_'):
```

Comment:
`escape_export_name` is used in two files, 
1. mongo.py `/Users/mac/devstack/edx-platform/common/lib/xmodule/xmodule/contentstore/mongo.py`
2. xml_importer.py `/Users/mac/devstack/edx-platform/common/lib/xmodule/xmodule/modulestore/xml_importer.py`

I guessed the location from the path `/Users/mac/devstack/edx-platform/common/lib/xmodule/xmodule` and put the method in the `util` folder. Also because that files highlights `Miscellaneous utility functions.`

Do we need to move it to any other location?